gcp/gke.ts

```diff
@@ -50,8 +51,23 @@ export class GcpCloud {
     });
 
     // separate node pool
-    new gcp.container.NodePool("primarynodes", config.gkeNodePoolArgs(cluster.name.get(), gcp.config.zone));
+    const nodePool = new gcp.container.NodePool(
+      util.name("primary-node-pool"),
+      {
+        ...config.gkeNodePoolArgs(cluster.name, gcp.config.zone),
+        version: engineVersion,
```

Comment:
This may look redundant, but it follows the official example:

https://github.com/pulumi/examples/blob/70d1d08e5322f6cc1b4178903cba47039e520284/gcp-ts-gke-hello-world/index.ts#L10-L44